docs/sources/reference/api/docker_remote_api.md

```diff
@@ -52,6 +52,21 @@ You can still call an old version of the API using
 You can now supply a `stream` bool to get only one set of stats and
 disconnect
 
+`GET /images/json`
+
+**New!**
+The `Created` field will be responsed as ISO 8601 String instead of UNIX epoch
```

Comment:
Perhaps something like this (for all three end points);

The `Created` field is now formatted as an ISO 8601 string instead of a UNIX timestamp, to be consistent with other parts of the API. **This is a breaking change**

But open to better suggestions (Should the breaking change be mentioned *here* or only in the release notes?)